src/main/java/dan200/computercraft/shared/common/BlockGeneric.java

```diff
@@ -219,12 +219,6 @@ public final AxisAlignedBB getBoundingBox( IBlockState state, IBlockAccess world
     }
 
     @Override
-    public final AxisAlignedBB getSelectedBoundingBox( IBlockState state, World world, BlockPos pos )
```

Comment:
My concern was small, but to do with clarity. When versions change it's good to see all the stuff you have to change in one place :)